src/inject.js

```diff
@@ -44,6 +45,24 @@ export default function injectDynamicImport(acorn) {
         return nextMethod.call(this, refDestructuringErrors);
       }
     ));
+
+    instance.extend('parseExpression', nextMethod => (
+      function parseExpression() {
+        const node = nextMethod.call(this);
+        if (node.type === 'CallExpression' && node.callee.type === 'Import') {
+          if (node.arguments.length !== 1) {
+            this.raise(node.start, 'import() requires exactly one argument');
+          }
+
+          const importArg = node.arguments[0];
+          if (importArg && importArg.type === 'SpreadElement') {
+            this.raise(importArg.start, '... is not allowed in import()');
+          }
+        }
+
+        return node;
+      }
+    ));
```

Comment:
Yeah that's fantastic. I'll adjust this to use that implementation! Thanks so much!